### PR TITLE
Kiali Iter8 Extension Installation Guide

### PR DESCRIPTION
**Pull request by jadeyliu939**
Adding Kiali Iter8 Extension installation steps, troubleshooting and Iter8-Kiali Features.

For [Issue#39](https://github.com/iter8-tools/docs/issues/39)

### CODE REVIEW

**Comment by fabolive** [comment]
We will want to submit this PR against the branch v0.2, which will subsume v0.1.

File: doc_files/kiali_install.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+```   
+bash <(curl -L https://kiali.io/getLatestKialiOperator) --accessible-namespaces '**' -oiv latest -kiv latest --operator-install-kiali true
+``` 
+
+will install the latest operator and Kiali from stable master.
+
+#### B. Enable iter_8 in Kiali Operator CR
+
+1. Follow the step [Create or Edit the Kiali CR](https://kiali.io/documentation/getting-started/#_create_or_edit_the_kiali_cr) or use ``kubectl edit kialis.kiali.io kiali -n kiali-operator`
```
Comment (Alan-Cha) [inline doc_files/kiali_install.md:21]: Extra backtick here

File: doc_files/kiali_install.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    #      enabled: false
+    #
+    extensions:
+      iter_8:
+        enabled: true
+    ```
+
+3. Make sure Kiali is restarted. Using `kubectl get pods -n istio-system` to check if Kiali pod is restarting
+
+4. Install [Iter8](https://github.com/iter8-tools/docs/blob/v0.1.0/doc_files/iter8_install.md)
```
Comment (Alan-Cha) [inline doc_files/kiali_install.md:43]: Is this the right version? `v0.1.0`?

File: doc_files/kiali_install.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+5. Start kiali - `istioctl dashboard kiali`
```
Comment (Alan-Cha) [inline doc_files/kiali_install.md:45]: Capitalize "kiali"?

**Pull request closed without merging**